### PR TITLE
WIP: RHDEVDOCS-5960 - JIRA link updates for GitOps releases 1.9.4, 1.10.2, and 1.11.1 

### PR DESCRIPTION
**Pull request by Dhruv-Soni11**
Version(s):

Must be CP'ed to GitOps 1.11, 1.10, and 1.9 releases

Issue:

https://issues.redhat.com/browse/RHDEVDOCS-5960

Link to docs preview:

https://73216--docspreview.netlify.app/openshift-gitops/latest/release_notes/gitops-release-notes

SME review: @reginapizza
QE review: @varshab1210
Peer review: @Srivaralakshmi

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by reginapizza** [comment]
LGTM but does since this has been changed to RHSA, does it need the type of heading in the doc that says the errata number? (like [here](https://docs.openshift.com/gitops/1.10/release_notes/gitops-release-notes.html#errata-updates-1-10.1_gitops-release-notes) in 1.10.1??

**Comment by Dhruv-Soni11** [comment]
> LGTM but does since this has been changed to RHSA, does it need the type of heading in the doc that says the errata number? (like [here](https://docs.openshift.com/gitops/1.10/release_notes/gitops-release-notes.html#errata-updates-1-10.1_gitops-release-notes) in 1.10.1?)

Yeah, I think once we get those details - we can add it here. WDYT @Srivaralakshmi?

**Comment by Srivaralakshmi** [comment]
> > LGTM but does since this has been changed to RHSA, does it need the type of heading in the doc that says the errata number? (like [here](https://docs.openshift.com/gitops/1.10/release_notes/gitops-release-notes.html#errata-updates-1-10.1_gitops-release-notes) in 1.10.1?)
> 
> Yeah, I think once we get those details - we can add it here. WDYT @Srivaralakshmi?

- 1.9.4 - RHSA-2024:0691-02
- 1.10.2 - RHSA-2024:0692-02
- 1.11.1 - RHSA-2024:0689-07

**Review by Srivaralakshmi — COMMENTED** [review]
@Dhruv-Soni11 Good work! Please add the errata updates section as discussed. Please keep the XX and NN parts in the links. Otherwise, the link will throw errors and PR builds won't pass.

The rest of the content LGTM. Thank you!

**Comment by Dhruv-Soni11** [comment]
> * 0689

Hi @Srivaralakshmi - I have added the content for the Errata updates. Please check.

**Review by Srivaralakshmi — COMMENTED** [review]
@Dhruv-Soni11 PTAL at my comment. The rest is LGTM. Please ping me when the PR is ready for another look. TY!

File: modules/gitops-release-notes-1-10-2.adoc [hunk 1]
```diff
@@ -8,12 +8,31 @@
...
+== Errata updates
+
+[id="rhsa-2024-0692-02-gitops-1-10-2-security-update-advisory_{context}"]
+=== RHSA-xxxx:nnnn - {gitops-title} 1.10.2 security update advisory
+
+Issued: 2024-03-15
+
+The list of security fixes that are included in this release is documented in the following advisory:
+
+* link:https://access.redhat.com/errata/RHSA-2023:5029[RHSA-xxxx:nnnn]
```
[the remark this answers is not in the document]
Comment (Dhruv-Soni11) [reply modules/gitops-release-notes-1-10-2.adoc]: Fixed it at this location and at all the places in the other files.

[the remark this answers is not in the document]
Comment (Srivaralakshmi) [reply modules/gitops-release-notes-1-10-2.adoc]: LGTM. thank you!

**Comment by reginapizza** [comment]
hey @Dhruv-Soni11, just checked on [this jira](https://issues.redhat.com/browse/CLOUDDST-22207) and in the comments you'll see I was able to confirm that the errata advisory numbers would stay the same as the previous release, so you should be able to use the same ones :)

**Comment by reginapizza** [comment]
@Srivaralakshmi can you explain where the `-02` and `-07` are coming from after the XXXX:nnnn? In our release notes previously it only shows the numbers as XXXX:nnnn without any numbers afterwards

**Comment by Srivaralakshmi** [comment]
> @Srivaralakshmi can you explain where the `-02` and `-07` are coming from after the XXXX:nnnn? In our release notes previously it only shows the numbers as XXXX:nnnn without any numbers afterwards

@reginapizza Of course. If you see in the errata title, it is mentioned as is with  `-02` and `-07`. Thank you!

**Comment by reginapizza** [comment]
@Srivaralakshmi the errata titles _where_ though? Everywhere I'm looking (like [here](https://access.redhat.com/errata-search/?q=&p=1&sort=portal_publication_date+desc&rows=10&portal_product=Red%5C+Hat%5C+OpenShift%5C+GitOps) for example) none of the errata titles have a format like that, and I don't see anywhere in our release notes for security advisories with anything like that.

**Comment by Srivaralakshmi** [comment]
> @Srivaralakshmi the errata titles _where_ though? Everywhere I'm looking (like [here](https://access.redhat.com/errata-search/?q=&p=1&sort=portal_publication_date+desc&rows=10&portal_product=Red%5C+Hat%5C+OpenShift%5C+GitOps) for example) none of the errata titles have a format like that, and I don't see anywhere in our release notes for security advisories with anything like that.

@reginapizza I am not sure whether it is okay to share the errata screenshot in GH. I have shared it in the slack. PTAL. TY!

File: modules/gitops-release-notes-1-10-2.adoc [hunk 2]
```diff
@@ -8,12 +8,31 @@
 
 {gitops-title} 1.10.2 is now available on {OCP} 4.12, 4.13, and 4.14.
 
+[id="errata-updates-1-10-2_{context}"]
+== Errata updates
+
+[id="rhsa-2024-0692-02-gitops-1-10-2-security-update-advisory_{context}"]
```
Comment (Srivaralakshmi) [inline modules/gitops-release-notes-1-10-2.adoc:14]:
```suggestion
[id="rhsa-2024-069-02-gitops-1-10-2-security-update-advisory_{context}"]
```

@Dhruv-Soni11 FYA.

Comment (Dhruv-Soni11) [reply modules/gitops-release-notes-1-10-2.adoc]: Fixed.

File: modules/gitops-release-notes-1-11-1.adoc [hunk 3]
```diff
@@ -8,12 +8,31 @@
 
 {gitops-title} 1.11.1 is now available on {OCP} 4.12, 4.13, and 4.14.
 
+[id="errata-updates-1-11-1_{context}"]
+== Errata updates
+
+[id="rhsa-2024-0689-07-gitops-1-11-1-security-update-advisory_{context}"]
```
Comment (Srivaralakshmi) [inline modules/gitops-release-notes-1-11-1.adoc:14]:
```suggestion
[id="rhsa-2024-0689-gitops-1-11-1-security-update-advisory_{context}"]
```

Comment (Srivaralakshmi) [reply modules/gitops-release-notes-1-11-1.adoc:14]: @Dhruv-Soni11 Fix this too, please!

File: modules/gitops-release-notes-1-9-4.adoc [hunk 4]
```diff
@@ -8,12 +8,31 @@
 
 {gitops-title} 1.9.4 is now available on {OCP} 4.12, 4.13, and 4.14.
 
+[id="errata-updates-1-9-4_{context}"]
+== Errata updates
+
+[id="rhsa-2024-0691-02-gitops-1-9-4-security-update-advisory_{context}"]
```
Comment (Srivaralakshmi) [inline modules/gitops-release-notes-1-9-4.adoc:14]:
```suggestion
[id="rhsa-2024-0691-gitops-1-9-4-security-update-advisory_{context}"]
```
@Dhruv-Soni11 also, this one!

Comment (Dhruv-Soni11) [reply modules/gitops-release-notes-1-9-4.adoc:14]: Fixed.

File: modules/gitops-release-notes-1-10-2.adoc [hunk 5]
```diff
@@ -8,12 +8,31 @@
 
 {gitops-title} 1.10.2 is now available on {OCP} 4.12, 4.13, and 4.14.
 
+[id="errata-updates-1-10-2_{context}"]
+== Errata updates
+
+[id="rhsa-2024-0692-gitops-1-10-2-security-update-advisory_{context}"]
+=== RHSA-2024-0692 - {gitops-title} 1.10.2 security update advisory
+
+Issued: 2024-03-15
```
Comment (varshab1210) [inline modules/gitops-release-notes-1-10-2.adoc:17]:
```suggestion
Issued: 2024-02-05
```
This is what the link says

File: modules/gitops-release-notes-1-11-1.adoc [hunk 6]
```diff
@@ -8,12 +8,31 @@
 
 {gitops-title} 1.11.1 is now available on {OCP} 4.12, 4.13, and 4.14.
 
+[id="errata-updates-1-11-1_{context}"]
+== Errata updates
+
+[id="rhsa-2024-0689-gitops-1-11-1-security-update-advisory_{context}"]
+=== RHSA-2024-0689 - {gitops-title} 1.11.1 security update advisory
+
+Issued: 2024-03-15
```
Comment (varshab1210) [inline modules/gitops-release-notes-1-11-1.adoc:17]:
```suggestion
Issued: 2024-02-05
```
This is what the link says

File: modules/gitops-release-notes-1-9-4.adoc [hunk 7]
```diff
@@ -8,12 +8,31 @@
 
 {gitops-title} 1.9.4 is now available on {OCP} 4.12, 4.13, and 4.14.
 
+[id="errata-updates-1-9-4_{context}"]
+== Errata updates
+
+[id="rhsa-2024-0691-gitops-1-9-4-security-update-advisory_{context}"]
+=== RHSA-2024-0691 - {gitops-title} 1.9.4 security update advisory
+
+Issued: 2024-03-15
```
Comment (varshab1210) [inline modules/gitops-release-notes-1-9-4.adoc:17]:
```suggestion
Issued: 2024-02-05
```
This is what the link says

File: modules/gitops-release-notes-1-11-1.adoc (hunk 6 above)
Comment (Dhruv-Soni11) [reply modules/gitops-release-notes-1-11-1.adoc]: Fixed.

File: modules/gitops-release-notes-1-9-4.adoc (hunk 7 above)
Comment (Dhruv-Soni11) [reply modules/gitops-release-notes-1-9-4.adoc]: Fixed.

File: modules/gitops-release-notes-1-10-2.adoc (hunk 5 above)
Comment (Dhruv-Soni11) [reply modules/gitops-release-notes-1-10-2.adoc]: Fixed.

**Comment by Dhruv-Soni11** [comment]
@reginapizza and @varshab1210 - Fixed the changes as per your inputs. PTAL and provide an LGTM if everything is ok.

**Comment by varshab1210** [comment]
LGTM

**Comment by reginapizza** [comment]
LGTM!

**Comment by mramendi** [comment]
Unfortunately, this does not work as a cp to GitOps 1.9 and 1.10. I will create the additional PRs myself.

**Comment by Dhruv-Soni11** [comment]
Closing this PR as content has already been merged by @mramendi in the following [PR](https://github.com/openshift/openshift-docs/pull/73653) as I was away on PTO.

**Pull request closed without merging**